agents/agent_engine/memory_bank/get_started_with_memory_bank.ipynb

```
@@ -0,0 +1,1856 @@
+      },
+      "outputs": [],
+      "source": [
+        "# Use the environment variable if the user doesn't provide Project ID.\n",
+        "import os\n",
+        "\n",
+        "import vertexai\n",
+        "\n",
+        "PROJECT_ID = \"[your-project-id]\"  # @param {type: \"string\", placeholder: \"[your-project-id]\", isTemplate: true}\n",
+        "if not PROJECT_ID or PROJECT_ID == \"[your-project-id]\":\n",
+        "    PROJECT_ID = str(os.environ.get(\"GOOGLE_CLOUD_PROJECT\"))\n",
+        "\n",
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)